packages/perseus/src/widgets/interactive-graphs/mafs-styles.css

```diff
@@ -203,6 +203,9 @@
         format("woff");
 }
 
+.MafsView pattern g {
+    stroke: #909296;
```

Comment:
Is this a Wonder Blocks color? How did we choose this particular shade?

Reply:
Yes! I spoke with Sarah B and she got me the equivalent hexcode for the value we are using for offBlack50, which "juuuust passes" the required 3:1 contrast for non-text content. 

Currently the wonderblocks colour uses transparencies (rgba) which makes the normally exported colour unusable in this case. I submitted a request to the wonderblocks team that we update all our colours to be hex codes as transparencies are a bad practice for brand colours / brand consistency.

Reply:
Wait! There are new `fadedOffBlack` colors that should solve this issue now!! @SonicScrewdriver

Reply:
You can see them allll the way at the bottom on the [Wonder Blocks Colors Storybook docs](https://khan.github.io/wonder-blocks/?path=/docs/packages-tokens-color--docs)